core/src/main/java/com/sequenceiq/cloudbreak/service/StackCommonService.java

```diff
@@ -254,7 +253,6 @@ public GeneratedBlueprintV4Response postStackForBlueprint(StackV4Request stackRe
         return response;
     }
 
-    @PreAuthorize("hasRole('AUTOSCALE')")
```

Comment:
why we removed this ?

Reply:
AutoscaleV4Controller::getCertificate already has @PreAuthorize("hasRole('AUTOSCALE')"). Hence this was duplicate check.